port/common/omrsysinfo.c

```diff
@@ -1178,3 +1178,17 @@ omrsysinfo_cgroup_subsystem_iterator_destroy(struct OMRPortLibrary *portLibrary,
 {
 	return;
 }
+
+/**
+ * Get the process start time in ns precision epoch time.
+ * @param[in] portLibrary The port library
+ * @param[in] pid The process ID
+ * @return 0 if the process does not exist, process start time in ns precision epoch time if the process exists
+ */
+uint64_t
+omrsysinfo_get_process_start_time(struct OMRPortLibrary *portLibrary, uintptr_t pid)
+{
+	Trc_PRT_sysinfo_get_process_start_time_enter((unsigned long long)pid);
+	Trc_PRT_sysinfo_get_process_start_time_exit((unsigned long long)pid);
```

Comment:
These tracepoint calls can be removed. This function is just a placeholder for the function description and will never be invoked since there is a implementation for each platform.
```suggestion
```